TKGame/TKGame.cs

```diff
@@ -60,17 +63,12 @@ protected override void Initialize()
 
             //Create New Background Object w/variables for setting Rectangle and Texture
             BackgroundImage = new Background(screenWidth, screenHeight, graphics.GraphicsDevice);
- 
+
             // TODO: Remove magic numbers
-            // We'll eventually probably want to generate walls based off level data (from a file).
```

Comment:
Since you are moving all the walls out of TKGame, you may want to add a way to change the background in the level editor in the future.